common/components/pagination/pagination.yaml

```diff
@@ -93,38 +86,33 @@ examples:
           text: "two"
         - href: "/page-3"
           text: "three"
-  - name: with items with class
```

Comment:
These were removed from the examples as each example is rendered in the components explorer. These two variations had no visual difference so that's why they've been removed from here and are now tested separately within the module test file.